### PR TITLE
CC-20: converted all a tags to Link tags

### PR DESCRIPTION
**Pull request by StephanieKirshner**

### CODE REVIEW

File: src/components/DetailPage/DetailPage.jsx [hunk 1]
```diff
@@ -82,16 +83,16 @@ const DetailPage = () => {
           </p>
         </div>
 
-        <a href='#' id='edit'>
+        <Link href='#' id='edit'>
```
Comment (jdickerson-banyan) [inline src/components/DetailPage/DetailPage.jsx:86]: Links do not use hrefs. Instead, the structure is this:
`<Link to="/about">About</Link>`
or if you do not know the path you can use `to="#"`

File: src/components/DetailPage/DetailPage.jsx [hunk 2]
```diff
@@ -82,16 +83,16 @@ const DetailPage = () => {
...
           Edit
-        </a>
-        <a href='#' id='delete'>
+        </Link>
+        <Link href='#' id='delete'>
```
Comment (jdickerson-banyan) [inline src/components/DetailPage/DetailPage.jsx:89]: to instead of href

File: src/components/DetailPage/DetailPage.jsx [hunk 3]
```diff
@@ -82,16 +83,16 @@ const DetailPage = () => {
...
           Delete
-        </a>
+        </Link>
         <br />
-        <a href='#' id='home'>
+        <Link href='#' id='home'>
```
Comment (jdickerson-banyan) [inline src/components/DetailPage/DetailPage.jsx:93]: href

**Review by jdickerson-banyan — CHANGES_REQUESTED** [review]

File: src/components/DetailPage/DetailPage.jsx (hunk 1 above)
Comment (StephanieKirshner) [reply src/components/DetailPage/DetailPage.jsx]: done

File: src/components/DetailPage/DetailPage.jsx (hunk 2 above)
Comment (StephanieKirshner) [reply src/components/DetailPage/DetailPage.jsx]: done

File: src/components/DetailPage/DetailPage.jsx (hunk 3 above)
Comment (StephanieKirshner) [reply src/components/DetailPage/DetailPage.jsx]: done

**Review by jdickerson-banyan — APPROVED** [review]

**Pull request merged**